backend/schemas/source/base/Base.libsonnet

```diff
@@ -129,75 +127,75 @@ local Enum = {
     title: 'EmptyString_EmptyArray_Null',
   },
   UnitedStatesStateAbbr: Types.string {
-     'enum': [
-        'AL',
-        'AK',
-        'AZ',
-        'AR',
-        'CA',
-        'CO',
-        'CT',
-        'DE',
-        'FL',
-        'GA',
-        'HI',
-        'ID',
-        'IL',
-        'IN',
-        'IA',
-        'KS',
-        'KY',
-        'LA',
-        'ME',
-        'MD',
-        'MA',
-        'MI',
-        'MN',
-        'MS',
-        'MO',
-        'MT',
-        'NE',
-        'NV',
-        'NH',
-        'NJ',
-        'NM',
-        'NY',
-        'NC',
-        'ND',
-        'OH',
-        'OK',
-        'OR',
-        'PA',
-        'RI',
-        'SC',
-        'SD',
-        'TN',
-        'TX',
-        'UT',
-        'VT',
-        'VA',
-        'WA',
-        'WV',
-        'WI',
-        'WY'
-     ],
+    enum: [
+      'AL',
```

Comment:
Unrelated to this PR but I came across this yesterday and would like your thoughts @tadhg-ohiggins @jadudm. Should not we add DC and U.S. territories to this list ?

Reply:
We've discussed it, and the issue is that we don't know what the actual full list of codes to accept is just yet. I think of this as part of the “Look at fields that might have standards, and move to standards.” item of the second chunk of #1140, which should probably be its own ticket at this point.